test/MUXControlsTestApp/master/CommonStylesVisualTreeTests_VerifyVisualTreeExampleForLightTheme-Light-7.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Element Type="Windows.UI.Xaml.Controls.Grid">
+  <Property Name="Padding" Value="0,0,0,0" />
```

Comment:
Just curious if it would make sense to generate the following and get rid of all the Element/Property/Values making it look closer to xaml.

<Windows.UI.Xaml.Controls.Grid
      Padding="0,0,0,0" 
      ...

Reply:
yes. now it look like this:
```
<Windows.UI.Xaml.Controls.Grid>
  Padding="0,0,0,0"
```